### ISSUE TITLE
Render to disk

### ISSUE THREAD

**Issue by goncalogato** [csound/web-ide#395]
Hello. Render to disk does not work (on Mac)... I can see the file in the WEB-IDE but can't really download the file... any solutions? thanks

**Comment by vlazzarini** [comment]
Works for me here, I am using Chrome. I can right click, select download. It then opens on a separate tab and from there I can download/save to my disk.

**Comment by goncalogato** [comment]
Using Safari, here. Have tried disabling the content blocker. No success. Instead of downloading the file, Safari opens a tab with the quicktime player. Can't save.

**Comment by vlazzarini** [comment]
yes, that's right a tab is opened but then on Chrome you download it from there. I guess you need to see why Safari doesn't let you download the file. In fact, it's not even download, it's just copying out of the browser sandbox. I actually tried this on my iphone with Safari and I can save it to the phone and open it in the files app.

**Comment by vlazzarini** [comment]
I just had a quick look, and Safari really doesn't seem to allow file downloading from the media viewer. I searched and the instructions seem to mention a context menu where "download video" would show up but it does not here. Saving as "page source" puts 0 bytes on disk and saving as webarchive works but you can only open it with safari again and nothing else. I tried an extractor and that did not work either.

At this point I'd just recommend using Chrome or Firefox.

**Comment by goncalogato** [comment]
Exactly my experience. Thanks.

**Comment by goncalogato** [comment]
Is this going to be sorted? Really enjoy the csound web-ide interface. Thanks

**Comment by vlazzarini** [comment]
one (unrelated) thing I noticed is that performance on Safari seems to be poor. Chrome and Firefox run the code way faster.
